### PR TITLE
add official bundles type definitions package

### PR DESCRIPTION
**Pull request by ori-shalom**

### CODE REVIEW

File: sdk/README.md [hunk 1]
```diff
@@ -14,6 +14,5 @@ This folder contains SDKs that connect to the Vault.
 
 1. [Vault Client](./vault-client) - client to connect to the Vault
 1. [Vault Encryption with TypeORM](./typeorm-encryption) - SDK to automatically encrypt/decrypt the TypeORM fields
-2. [Vault PCI Forms](./forms) - processes sensitive data without exposing it to your systems by automatically tokenizing it before submitting it to your backend. Typical use case is for processing credit card numbers and still remaining outside of a PCI scope
-
-
+1. [Vault PCI Forms](./forms) - processes sensitive data without exposing it to your systems by automatically tokenizing it before submitting it to your backend. Typical use case is for processing credit card numbers and still remaining outside of a PCI scope
+1. [Vault Bundles](./bundles) - Type definitions for creating and using Vault bundles
```
Comment (nirhaas) [inline sdk/README.md:18]: Can you link to the bundle docs here as well?

File: sdk/bundles/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "name": "@piiano/bundles",
+  "version": "0.0.1",
+  "description": "Piiano Vault type definitions for bundles",
+  "types": "dist/index.d.ts",
+  "author": "Ori Shalom <ori.shalom@piiano.com>",
+  "files": [
+    "dist"
+  ],
+  "keywords": [
+    "Piiano",
```
Comment (nirhaas) [inline sdk/bundles/package.json:11]: is that case sensitive?

**Review by nirhaas — APPROVED** [review]
Approved but please see my question in Slack before merging

File: sdk/bundles/package.json (hunk 2 above)
Comment (ori-shalom) [reply sdk/bundles/package.json:11]: Looks like both
Capitalized https://www.npmjs.com/search?q=keywords%3APiiano
and
Lowercase https://www.npmjs.com/search?q=keywords%3Apiiano
yields the same results.

File: examples/custom-data-types/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import type {VaultFunction} from "@piiano/vault-bundles-types";
+
```
Comment (david-piiano) [inline examples/custom-data-types/src/index.ts:2]: Add comment about older Vault that did not accept these SSNs

Comment (david-piiano) [reply examples/custom-data-types/src/index.ts:2]: @ori-shalom something like: "Vaults prior to 1.10.4 had a stricter SSN validation. In this example we provided a new data type that has relaxed SSN validation that can work with earlier vaults."

**Pull request merged**